ui/components/RunnableCodeExample.gd

```diff
@@ -313,11 +326,12 @@ func _set_scene_instance(new_scene_instance: CanvasItem) -> void:
 				rect.position += offset
 				rect.size.x = (rect.size.x * variable_name.length()) + 4
 
-				var monitored_variable: CodeExampleVariableUnderline = CodeExampleVariableUnderlineScene.instance()
```

Comment:
I invite you to use the formatter gdformat. Without space is the official gdscript style guide